### PR TITLE
#167326132 Refactored status messages

### PR DESCRIPTION
**Pull request by hezronkimutai**
#### What does this PR do?
This PR creates a pull fixes all the status messages
#### Description of Task to be completed
- Fix all the status messages as per the ADC
#### How should this be manually tested?
- Clone this repo
- Checkout to branch bg-status-messages-167326132
- Use postman to test all the endpoint and observe the status messages

#### Any background context you want to provide?
- With some edge cases that I haven't captured yet, I haven't implemented the edge cases.
#### What are the relevant pivotal tracker stories?
[#167326132](https://www.pivotaltracker.com/n/projects/2353827/stories/167326132)

### CODE REVIEW

File: api/v2/controllers/properties.js [hunk 1]
```diff
@@ -14,44 +14,49 @@ const postPropertiesController = async(res, inputs) => {
...
+    }catch(err){
+      console.log("---------------------",err)
+    }
+    
   }
 }
 const getPropertiesController = async(res, req) => {
-  const propertiesQuery = format(`SELECT * from properties`)
+  const propertiesQuery = `SELECT * from properties`
   db.query(propertiesQuery, function (err, result) {
```
Comment (TheoOkafor) [inline api/v2/controllers/properties.js:40]: use ES6

File: api/v2/controllers/properties.js [hunk 2]
```diff
@@ -14,44 +14,49 @@ const postPropertiesController = async(res, inputs) => {
...
+        }
+        res.status(201).json({
+          status: '201',
+          message: 'Property created Succesfully',
+          data:inputs
+        })
       })
-    })
+    }catch(err){
+      console.log("---------------------",err)
```
Comment (TheoOkafor) [inline api/v2/controllers/properties.js:33]: no unnecessary `console.log` statements

**Review by TheoOkafor — CHANGES_REQUESTED** [review]
Attend to these comments below
